docs/user-guide/mvd-3270.md

```diff
@@ -0,0 +1,120 @@
+# 3270 Terminal
+The 3270 Terminal Display Emulator plugin provides a user interface that emulates the basic functions of IBM 3270 family terminals. On the "back end," the plugin and the Zowe Application Server connect to any standard TN3270/E server.
+
+This terminal display emulator operates as a "Three-Tier" program. Due to web browsers being unable to supply TCP networking that terminals require, this terminal display emulator does not connect directly to your TN3270 server. Instead, the Zowe Application Server acts as a bridge, and uses websockets between it and the browser for terminal communication. As a result, terminal connections only work when the stack of network programs supports websockets and the TN3270 server destination is visible to the Zowe Application Server.
+
+The terminal connection can be customized per-user and saved for future sessions using the connection toolbar of the application. The preferences are stored within [the configuration dataservice storage](../extend/extend-desktop/mvd-configdataservice), which can also be used to set instance-wide defaults for multiple users.
+
+## Keyboard shortcuts
+The terminal bundle that is used by tn3270-ng2 has several keyboard shortcuts to execute common TN3270 emulator functionality. Below is the current list of what function is mapped to which key combination. 
+
+Be aware that some browsers intercept key combinations that are common to browsers, such as Control+R or Control+T, and that the OS may also intercept certain combinations such as Windows+R or Alt+F4. 
```

Comment:
- our style for keyboard keys is to use backticks to format keys: `ctrl` + `R`
- please write out OS